CMakeLists.txt

```diff
@@ -104,6 +104,9 @@ ELSE()
   MESSAGE(STATUS "Authentication Plugins installation dir: ${INSTALL_PLUGINDIR}")
 ENDIF()
 
+IF(UNIX AND NOT WITH_SSL)
+  SET(WITH_SSL "OPENSSL")
+ENDIF()
 IF(WIN32 OR WITH_OPENSSL OR "${WITH_SSL}" STREQUAL  "OPENSSL")
```

Comment:
I think this isn't quite correct, in fact, this `CMakeLists.txt` should not do anything with `WITH_SSL`, should not interpret its values, add libraries, nothing. C/ODBC does not need an SSL library. C/C libmariadb does. But it has its own `CMakeLists.txt` that validates `WITH_SSL` and links with correct libraries, etc. So C/ODBC doesn't need to.

Reply:
ok, removed my change